repos/system_upgrade/el7toel8/actors/excluderepositories/libraries/excluderepositories.py

```diff
@@ -0,0 +1,145 @@
+from leapp import reporting
+from leapp.libraries.common.config import get_product_type
+from leapp.libraries.stdlib import api
+from leapp.models import (
+    CustomTargetRepository,
+    RepositoriesExcluded,
+    RepositoriesFacts,
+    RepositoriesMap,
+)
+
+manually_enabled_repos_in_use = []
+
+
+def _report_using_unsupported_repos(repo_ids):
+    if manually_enabled_repos_in_use:
```

Comment:
```suggestion
    if repo_ids:
```
Since you call the function this way `_report_using_unsupported_repos(manually_enabled_repos_in_use)`, the `repo_ids` == `manually_enabled_repos_in_use`.